loops/loop44.go

```diff
@@ -0,0 +1,21 @@
+// Convert lowercase string to upper case 
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	source := "hello world"
+	destination := ""
+	for _, v := range source {
```

Comment:
strings.ToUpper

Reply:
You mean i should use string.ToUpper method. ? , i wanted to avoid that explicitly here to understand better.

Reply:
Ok, in that case print the source again using string.ToUpper method as well.